docs/mint.json

```diff
@@ -146,6 +147,207 @@
         }
       ]
     },
+    {
+      "group": "FHIR",
+      "pages": [
+        {
+          "group": "Data Types",
+          "pages": [
+            "medical-api/fhir/data-types/address",
+            "medical-api/fhir/data-types/age",
+            "medical-api/fhir/data-types/annotation",
+            "medical-api/fhir/data-types/attachment",
+            "medical-api/fhir/data-types/codeableconcept",
+            "medical-api/fhir/data-types/coding",
+            "medical-api/fhir/data-types/contactdetail",
+            "medical-api/fhir/data-types/contactpoint",
+            "medical-api/fhir/data-types/contributor",
+            "medical-api/fhir/data-types/count",
+            "medical-api/fhir/data-types/datarequirement",
+            "medical-api/fhir/data-types/distance",
+            "medical-api/fhir/data-types/dosage",
+            "medical-api/fhir/data-types/duration",
+            "medical-api/fhir/data-types/elementdefinition",
+            "medical-api/fhir/data-types/expression",
+            "medical-api/fhir/data-types/humanname",
+            "medical-api/fhir/data-types/identifier",
+            "medical-api/fhir/data-types/marketingstatus",
+            "medical-api/fhir/data-types/meta",
+            "medical-api/fhir/data-types/money",
+            "medical-api/fhir/data-types/moneyquantity",
+            "medical-api/fhir/data-types/narrative",
+            "medical-api/fhir/data-types/parameterdefinition",
+            "medical-api/fhir/data-types/period",
+            "medical-api/fhir/data-types/population",
+            "medical-api/fhir/data-types/prodcharacteristic",
+            "medical-api/fhir/data-types/productshelflife",
+            "medical-api/fhir/data-types/quantity",
+            "medical-api/fhir/data-types/range",
+            "medical-api/fhir/data-types/ratio",
+            "medical-api/fhir/data-types/reference",
+            "medical-api/fhir/data-types/relatedartifact",
+            "medical-api/fhir/data-types/sampleddata",
+            "medical-api/fhir/data-types/signature",
+            "medical-api/fhir/data-types/simplequantity",
+            "medical-api/fhir/data-types/substanceamount",
+            "medical-api/fhir/data-types/timing",
+            "medical-api/fhir/data-types/triggerdefinition",
+            "medical-api/fhir/data-types/usagecontext"
+          ]
+        },
+        {
+          "group": "Resources",
+          "pages": [
```

Comment:
Did you generate this list and "Data Types" automatically or had to build it by hand? Curious about the process, that's all. :)

Reply:
Kind of, it was manual but i used copilot which helped speed it up with suggestions